### PR TITLE
feature: add static display config when running with a systemd session

### PR DESCRIPTION
**Pull request by mattkae**

### CODE REVIEW

File: src/miracle-wm-session.systemd.in [hunk 1]
```diff
@@ -1,2 +1,4 @@
 #!/bin/sh
-systemd-cat --identifier=miracle-wm @CMAKE_INSTALL_FULL_BINDIR@/miracle-wm --systemd-session-configure=@CMAKE_INSTALL_FULL_LIBEXECDIR@/miracle-wm-session-setup
+systemd-cat --identifier=miracle-wm @CMAKE_INSTALL_FULL_BINDIR@/miracle-wm \
+    --systemd-session-configure=@CMAKE_INSTALL_FULL_LIBEXECDIR@/miracle-wm-session-setup \
+    --display-config=static=$(systemd-path user-configuration)/miracle-wm-display.yaml
```
Comment (Conan-Kudo) [inline src/miracle-wm-session.systemd.in:4]: What happens when the file doesn't exist?

Comment (mattkae) [reply src/miracle-wm-session.systemd.in:4]: It gets created with with a default configuration that includes all currently connected outputs :)

Comment (Conan-Kudo) [reply src/miracle-wm-session.systemd.in:4]: Answered in chat, it'll create it.

Comment (mattkae) [reply src/miracle-wm-session.systemd.in:4]: e.g.:

```yaml
layouts:
# keys here are layout labels (used for atomically switching between them).
# The yaml anchor 'the_default' is used to alias the 'default' label

  default:
    cards:
    # a list of cards (currently matched by card-id)

    - card-id: 0
      unknown-1:
        # This output supports the following modes: 1280x1024@60.0
        #
        # Uncomment the following to enforce the selected configuration.
        # Or amend as desired.
        #
        state: enabled	# {enabled, disabled}, defaults to enabled
        mode: 1280x1024@60.0	# Defaults to preferred mode
        position: [0, 0]	# Defaults to [0, 0]
        orientation: normal	# {normal, left, right, inverted}, defaults to normal
        scale: 1
        group: 0	# Outputs with the same non-zero value are treated as a single display

  side_by_side:
    cards:
    # a list of cards (currently matched by card-id)

    - card-id: 0
      unknown-1:
        # This output supports the following modes: 1280x1024@60.0
        #
        # Uncomment the following to enforce the selected configuration.
        # Or amend as desired.
        #
        state: enabled	# {enabled, disabled}, defaults to enabled
        mode: 1280x1024@60.0	# Defaults to preferred mode
        position: [0, 0]	# Defaults to [0, 0]
        orientation: normal	# {normal, left, right, inverted}, defaults to normal
        scale: 1
        group: 0	# Outputs with the same non-zero value are treated as a single display
```

Comment (mattkae) [reply src/miracle-wm-session.systemd.in:4]: (that's for my X11-hosted, so it says "unknown" instead of the display name. but this is the general idea)

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]
Typo in the commit message: "runnign" -> "running"

**Pull request merged**